src/test/functional/queryServices.test/tags.integration.ts

```diff
@@ -365,6 +368,164 @@ describe('tags query tests - happy path', () => {
         .hasPreviousPage
     ).is.false;
   });
+
+  it('return list of SuggestedTags and paginated savedItems for SavedItem', async () => {
+    await db('readitla_b.item_grouping').insert([
+      {
+        resolved_id: 1,
+        grouping_id: 10631688,
+        source_score: 1.0,
+      },
+      {
+        resolved_id: 1,
+        grouping_id: 10631689,
+        source_score: 1.0,
+      },
+    ]);
+
+    await db('suggested_tags_user_grouping_tags').insert([
+      {
+        user_id: 1,
+        grouping_id: 10631688,
+        tag: 'zebra',
+        weighted_count: 0.9211,
+        count: 2,
+      },
+      {
+        user_id: 1,
+        grouping_id: 10631689,
+        tag: 'travel',
+        weighted_count: 0.9211,
+        count: 2,
+      },
+    ]);
+
+    await db('suggested_tags_user_groupings').insert([
+      {
+        user_id: 1,
+        grouping_id: 10631688,
+        weighted_count: 1.6461,
+        count: 1,
+      },
+      {
+        user_id: 1,
+        grouping_id: 10631689,
+        weighted_count: 1.6461,
+        count: 1,
+      },
+    ]);
+
+    const variables = {
+      userId: '1',
+      itemId: '1',
+    };
+
+    const GET_SUGGESTED_TAGS_FOR_SAVED_ITEM = gql`
+      query getSavedItem($userId: ID!, $itemId: ID!) {
+        _entities(representations: { id: $userId, __typename: "User" }) {
+          ... on User {
+            savedItemById(id: $itemId) {
+              url
+              suggestedTags {
+                ... on Tag {
+                  id
+                  name
+                  _createdAt
+                  _updatedAt
+                  _version
+                  _deletedAt
+                  savedItems {
```

Comment:
do we care about the `savedItems` for suggested tags? If not, we have test coverage for `savedItems` under tags and we don't need to repeat them.

Reply:
i copy pasted :)

Reply:
You know what. I decided to leave it. It validates that I didnt break things for suggested tags